test/scripts/timeout

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
```

Comment:
Is this for cross-platform compatibility? My platform has `timeout`. https://www.gnu.org/software/coreutils/manual/html_node/timeout-invocation.html#timeout-invocation

Reply:
Ah, checked on my Mac and it's not there.